### PR TITLE
Task/Section_16: added summary, essay/praktikum, and screenshots

### PR DESCRIPTION
**Pull request by gozzafadillah**
- [x] added summary section 16 react form
- [x] added essay
1. Made validation with form controlled component
2. Made regex validation for name and email

- [x] added screenshots from my essay

### CODE REVIEW

File: 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+import React, { useState, useRef } from "react";
+import NavbarUtility from "../../utility/Navbar/NavbarUtility";
+import { Col, Container, Row } from "react-bootstrap";
+import FloatingLabel from "react-bootstrap/FloatingLabel";
+
+import "./form.css";
+import { useNavigate } from "react-router-dom";
+
+const FormComponent = () => {
+  const Navigated = useNavigate();
```
Comment (syukronarie) [inline 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: please funtion itu pake camelCase ya, yg pake PascalCase itu Instance, Class, Component

File: 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+  const baseError = {
+    nama: "",
+    email: "",
+    phoneNumber: "",
+  };
+
+  const suratKesungguhan = useRef(null);
+  const [data, setData] = useState(baseData);
+  const [errorMessage, setErrorMessage] = useState(baseError);
```
Comment (syukronarie) [inline 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: errorMessage ini utk banyak kan, baiknya tambahin s diakhir, krn jamak

**Review by syukronarie — APPROVED** [review]

File: 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx (hunk 1 above)
Comment (gozzafadillah) [reply 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: done mas

File: 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx (hunk 2 above)
Comment (gozzafadillah) [reply 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: done mas

File: 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx (hunk 1 above)
Comment (syukronarie) [reply 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: bukan function declarationnya mas bro, tapi `Navigated` ini function bukan instance

Comment (gozzafadillah) [reply 16_React_Form/praktikum/essay_react_form/src/components/Form/FormComponent.jsx]: walah salah liat

**Pull request merged**